angular-client/src/components/current-display/current-display.component.html

```diff
@@ -0,0 +1,9 @@
+<info-background title="Current" icon="Bolt">
+  <hstack style="width: 40%">
+    <div class="battery-display-body">
+      <typography variant="info-value" [content]="amps.toString()" />
+      <typography variant="info-unit" content="amps" />
+    </div>
+    <current-graph style="width: 60%; padding-bottom: 5vh"></current-graph>
```

Comment:
clean up the spacing here a bit, so it's not quite so crammed in (should ideally look kinda similar to the state of charge spacing around the graph)

Reply:
![image](https://github.com/Northeastern-Electric-Racing/Argos/assets/29521172/bc6d7540-7bf1-4273-b13b-b18ead114cfb)

Reply:
same spacing comment here
![image](https://github.com/Northeastern-Electric-Racing/Argos/assets/29521172/beddab28-c089-430e-be8f-75c96a754318)